pubsub/schemas/create_avro_schema.go

```diff
@@ -28,6 +28,7 @@ import (
 func createAvroSchema(w io.Writer, projectID, schemaID, avscFile string) error {
 	// projectID := "my-project-id"
 	// schemaID := "my-schema"
+	// For more info about the avro schemas see: https://cloud.google.com/pubsub/docs/schemas#avro
```

Comment:
This link doesn't take me to a specific section on this page, where should it point it?

Since the sample will appear on https://cloud.google.com/pubsub/docs/schemas, is it necessary to add this comment?

Reply:
Hm, that's fair. I added this because of Seth's comment about protos (how do you specify a proto schema?). Do we have a guide to writing avro/proto schemas and if not, I'll probably remove this comment.

Reply:
Per discussion on how-to guide, we'll probably create a blogpost/tutorial at some point in the future but for now, this feature is targeted at the majority of users who are already using schemas so this should not be new.